app/test_main.py

```diff
@@ -1,3 +1,14 @@
+import pytest
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize("cat_age, dog_age, expected", [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (28, 28, [3, 2]),
+])
+def test_get_human_age(cat_age: int, dog_age: int, expected: list) -> int:
+    assert get_human_age(cat_age, dog_age) == expected
```

Comment:
Add test for this case 
![image](https://github.com/mate-academy/py-cat-and-dog-years/assets/93856423/ebcd1089-4637-4f20-a881-c2c078ae3178)